helpers/tf_cfg.py

```diff
@@ -240,9 +240,10 @@ def v_level():
     return int(cfg.get("General", "Verbose"))
 
 
-def dbg(level, *args, **kwargs) -> None:
+def dbg(level: Union[int, str], msg: str, *args, **kwargs) -> None:
```

Comment:
Maybe we should only use `int` here?
```suggestion
def dbg(level: int, msg: str, *args, **kwargs) -> None:
```

Reply:
Yes, we can, I implemented as str and int because there was casting to int in an initial implementation and I thought it may receive string value from somewhere.